DirectProgramming/DPC++FPGA/Tutorials/Features/fp_pragmas/src/fp_pragmas.cpp

```diff
@@ -0,0 +1,259 @@
+//==============================================================
+// Copyright Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+#include <CL/sycl.hpp>
+#include <CL/sycl/INTEL/fpga_extensions.hpp>
+#include <iomanip>
+#include <iostream>
+#include <vector>
+
+// dpc_common.hpp can be found in the dev-utilities include folder.
+// e.g., $ONEAPI_ROOT/dev-utilities//include/dpc_common.hpp
+#include "dpc_common.hpp"
+
+using namespace sycl;
+using namespace std;
+
+#pragma clang fp contract(off)
```

Comment:
Comment here to indicate why you are applying these 'off' pragmas here and what it means for the tutorial.